Ez namey.js

```diff
@@ -0,0 +1,22 @@
+let firstWords = ["Innovative",
```

Comment:
All variables in this exercise can be defined as `const` as they are not reassigned.